### PR TITLE
Add base tests \o/ and autopep8 (fixes #98, addresses #51)

### LINKED ISSUE #98: PEP8 - generated songs don't fit standards
Generated songs don't fit PEP8 standard.
We should either don't test PEP8 on these files because we use them as generated files and overwrite them. Or make them PEP8 Compliant

### PR DESCRIPTION
**Pull request by simahawk**
@guewen @leemannd check this out! :smile:

### CODE REVIEW

File: setup/base_dj/setup.py [hunk 1]
```diff
@@ -2,7 +2,5 @@
 
 setuptools.setup(
     setup_requires=['setuptools-odoo'],
-    odoo_addon={'odoo_version_override': '10.0'},
-    include_package_data=True,
-    package_data={'': ['*', ]}
+    odoo_addon=True,
```
Comment (leemannd) [inline setup/base_dj/setup.py:5]: Question: we are not forcing anymore the version 10.0 of Odoo?

**Review by leemannd — APPROVED** [review]
Yeah! Thanks
I just have a personal comprehension question.

File: setup/base_dj/setup.py (hunk 1 above)
Comment (simahawk) [reply setup/base_dj/setup.py:5]: is not needed unless you want to override it. I was playing w/ setuptools-odoo w/ try n catch and I got this garbage. See https://github.com/OCA/website-cms/pull/34

**Comment by simahawk** [comment]
TODO: tests should also verify that we are really py-linted

File: setup/base_dj/setup.py (hunk 1 above)
Comment (leemannd) [reply setup/base_dj/setup.py:5]: Ok, thanks for the information

**Comment by simahawk** [comment]
I'm gonna include pylint validation in anther PR. Will come from this WIP branch https://github.com/camptocamp/odoo-dj/tree/dev

**Pull request merged**